lib/web_ui/dev/test_runner.dart

```diff
@@ -582,3 +647,17 @@ void _copyTestFontsIntoWebUi() {
     sourceTtf.copySync(destinationTtfPath);
   }
 }
+
+/// This objest is used as an input message to the PoolResources that are
+/// building the tests.
```

Comment:
nit: s/the tests/a test/ (an instance of this class only represents one test)

Reply:
done